### DISCUSSION TITLE
How to sync latest master with the project that using sassfly?

### DISCUSSION THREAD

**Discussion by yalcinos** [nextify-limited/saasfly#87] [Q&A]
Hi, I have been using sassfly and I liked the framework you guys provides. 

I see that you guys do some improvement and push to master. Since I already installed sassfly with bun. How can I get the latest version of master that sync with my current project? 

Other question is that when I navigate to Dashboard, Settings settings by clicking button there are 3-4 seconds delay. Do you know why it could be happening ?

**Comment by jackc3** [comment]
Hi @yalcinos !
Since Saasfly has not yet provided a stable version, we may make some upgrades that are not backwards compatible. 

Recently, we have made changes to the UI of the marketing page, the file path is:
<img width="533" alt="image" src="https://github.com/user-attachments/assets/8c35f7d3-5971-46a7-bbb5-ca81a12f37a0">

Other changes include upgrading Nextjs, tRPC and other frameworks.

So, I think you can copy another `apps/nextjs/src/app/[lang]/(marketing)/page.tsx` page. For example, copy `apps/nextjs/src/app/[lang]/(marketing)/page.tsx` to `apps/nextjs/src/app/[lang]/(marketing)/page_v2.tsx` .

And then `git pull` from master, this will recover your `apps/nextjs/src/app/[lang]/(marketing)/page.tsx` page.

Then, copy all the code from page_v2 to page.

I think this might be useful.

**Reply by yalcinos** [reply]
@jackc3 Thank you for your response. I am actually overriding whole design. So there is no problem with components sync. 
This is more about how are we going to sync the package updates. For example you guys upgrading Nextjs, tRPC and other frameworks. But my package json has the old version. I guess I need to update packages manually by grabbing from master, right?

Because I am getting this error and I guess you guys fixed by upgrading above packages.
<img width="700" alt="image" src="https://github.com/user-attachments/assets/7b02f75a-34f9-43c2-89c6-59fa5ceabaf9">

**Reply by jackc3** [reply]
Yes, you can update packages manually

**Comment by jackc3** [comment]
_3-4 seconds delay when you navigate to Dashboard_

Does this happen on your local computer?

**Reply by yalcinos** [reply]
Yes, it happens in local computer. When ever I click a button on page. It takes some time to navigate to another route usually 2-4 seconds. And I am using Macbook Pro m2 chip 32GB ram PC

**Reply by jackc3** [reply]
I can't reproduce the problem. Let me find out the reason.

**Reply by yalcinos** [reply]
Thanks @jackc3
